CDCgov/ocio-wsl
# fix: updating to the latest supported tools version

## Updates

- Updating dependencies based on `asdf latest --all` command.

## Testing Steps

- I built it locally, and it builds.
- Also ran `bash test.sh` - which runs `podman run -t localhost/ubuntu-22.04-cdc bash -c "bash /opt/scripts/add-extra-tools.sh"` to install all the tools within the image to ensure it installs properly.
